source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc

```diff
@@ -263,14 +263,16 @@ std::string SPIFFEValidator::extractTrustDomain(const std::string& san) {
   return "";
 }
 
-size_t SPIFFEValidator::daysUntilFirstCertExpires() const {
+absl::optional<size_t> SPIFFEValidator::daysUntilFirstCertExpires() const {
   if (ca_certs_.empty()) {
-    return 0;
+    return absl::make_optional(0);
```

Comment:
In `ContextManagerImpl::daysUntilFirstCertExpires()` we seem to return an optional with a very large value if there's no context (and therefore no certificates). We does this return 0?

Reply:
Honestly, I just keep it as it was.

Reply:
Maybe we can unite them.